Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -1087,6 +1087,11 @@ def parse_folder_as_json(folder):     # pragma: no cover
         if value:
             raw_dict[list_dict_field] = map(lambda x: parse_object_as_dict(x), value)
 
+    for list_str_field in ["categories"]:
```

Comment:
why for loop?

Reply:
For Future support, in the case of more entries lists of strings